onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -47,30 +47,43 @@
 
         if (contextPrivate->IsDmlGraphNode())
         {
-            // Create an edge list using sentinels for unused edges, as required by the SetDmlOperator ABI
-            auto ReplaceUnusedEdgeIndicesWithSentinel = [](gsl::span<const std::optional<uint32_t>> indices)
+            MLOperatorGraphDesc operatorGraphDesc = {};
+            operatorGraphDesc.nodeCount = 1;
+            std::vector<const DML_OPERATOR_DESC*> opDescs({&operatorDesc});
```

Comment:
>opDescs [](http://example.com/codeflow?start=50&length=7)

I don't see any additional elements being append to this vector. Does it need to be dynamic, if it's only ever size 1?

Reply:
Yes, it will always be size 1. I will use array instead.